packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -100,14 +101,19 @@ class PopupMenuDivider extends PopupMenuEntry<Never> {
   /// Creates a horizontal divider for a popup menu.
   ///
   /// By default, the divider has a height of 16 logical pixels.
-  const PopupMenuDivider({ Key? key, this.height = _kMenuDividerHeight }) : super(key: key);
+  const PopupMenuDivider({ Key key, this.height = _kMenuDividerHeight, this.color = _kMenuDividerColor })
+      : super(key: key);
 
   /// The height of the divider entry.
   ///
   /// Defaults to 16 pixels.
   @override
   final double height;
 
+  /// The color of the divider if it's selected.
+  @override
+  final Color? color;
```

Comment:
This doc does not seems right, this color is not applied if the divider is selected.

You can check the original comment from the Divider widget to see when it will be used
https://github.com/flutter/flutter/blob/a83755763499472053b5af8acc0f53b8727a4826/packages/flutter/lib/src/material/divider.dart#L149